src/w3c/headers.js

```diff
@@ -665,6 +665,12 @@ function derivePubSpace(conf) {
       return `/Submission`;
   }
 
+  if (conf.groupType === "other" &&
+    (conf.group === "tag" || conf.group === "ab") &&
+    conf.canonicalURI === "TR") {
```

Comment:
Can we require `conf.canonicalURI === "TR"` (at line 653) for all those who want to opt-in, not just TAG/AB (skipping group/groupType checks here)?